crime-applications-adaptor/application/build.gradle

```diff
@@ -24,7 +24,7 @@ def versions = [
         resilience4jVersion    : "2.2.0",
         commonsioVersion       : '2.11.0',
         sentryVersion          : "7.8.0",
-        commonsVersion         : "0.12.0",
+        commonsVersion: "1.2.1",
```

Comment:
Minor, can you tidy up the whitespace formatting so the : aligns?

Reply:
Spotless Gradle plugin is reformatting this if we align the format.